testgrid/config/config.yaml

```diff
@@ -874,6 +874,9 @@ test_groups:
 # azure
 - name: kubernetes-azure
   gcs_prefix: kube_azure_log/kubernetes.azure.nightly
+# Canonical Distribution of Kubernetes (contact: @chuckbutler on github)
+- name: canonical-kubernetes-e2e-gce
+  gcs_prefix: canonical-kubernetes-tests/gce/logs/e2e-node
```

Comment:
good shout. I updated the path to reflect what is in buckets.yaml.  I think the tree formula:

gss://bucket-name/logs/{cloud}/{test-profile}/

will work cleanly for this purpose. Thanks for calling this out.